app/tags.yaml

```diff
@@ -2,12 +2,174 @@
+    label on GitHub.
+Sharing:
+  description: Corresponds to the [Feature:Sharing](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ASharing)
+    label on GitHub.
+Shortcuts:
+  description: Top Sites on the Focus home page. Corresponds to the [Feature:Shortcuts](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AShortcuts)
+    label on GitHub.
+SitePermissions:
+  description: Corresponds to the [Feature:SitePermissions](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ASitePermissions)
+    label on GitHub.
+Snackbar:
+  description: Corresponds to the [Feature:Snackbar](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ASnackbar)
+    label on GitHub.
+Sync:
+  description: Corresponds to the [Feature:Sync](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ASync)
+    label on GitHub.
+SyncTabs:
+  description: Corresponds to the [Feature:SyncTabs](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ASyncTabs)
+    label on GitHub.
+Tabs:
+  description: Corresponds to the [Feature:Tabs](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ATabs)
+    label on GitHub.
+Telemetry:
+  description: Corresponds to the [Feature:Telemetry](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ATelemetry)
+    label on GitHub.
+TextSelection:
+  description: Corresponds to the [Feature:TextSelection](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ATextSelection)
+    label on GitHub.
+Themes:
+  description: Dark mode, light mode. Corresponds to the [Feature:Themes](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AThemes)
+    label on GitHub.
+Toolbar:
+  description: Address bar, see also Feature:Search. Corresponds to the [Feature:Toolbar](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AToolbar)
+    label on GitHub.
+TrackingProtection:
+  description: Corresponds to the [Feature:TrackingProtection](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3ATrackingProtection)
+    label on GitHub.
+Voice:
+  description: Corresponds to the [Feature:Voice](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AVoice)
+    label on GitHub.
+WebExtensions:
+  description: Corresponds to the [Feature:WebExtensions](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AWebExtensions)
+    label on GitHub.
 performance:
   description: Used for data reviews to label metrics related to performance. Corresponds
-    to the [Feature:Performance](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3APerformance)
+    to the [Feature:performance](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3Aperformance)
```

Comment:
Seeing that all other tags use an uppercase can we rename this tag on Github to also use uppercase?
(Then the script will have to be run again to ensure `Performance` remain as a tag)

Reply:
Done